### PR TITLE
Master

### PR DESCRIPTION
**Pull request by rwwh**
Another attempt, now with references to documents on the web. My github experience is exhausted so I hope this will work.

### CODE REVIEW

**Comment by danielskatz** [comment]
The GitHub part of this is fine, but I still have some issues with the context you want to add.

I will put specific comments in the PR.

File: README.md [hunk 1]
```diff
@@ -43,34 +43,35 @@ Evidence: Citations
...
 
+###Software is available in a human readable and separate computer readable form, data in only one of the two.
```
Comment (danielskatz) [inline README.md:48]: Data could also be in two forms, for example, ASCII and binary, or ASCII and something compressed.

File: README.md [hunk 2]
```diff
@@ -43,34 +43,35 @@ Evidence: Citations
...
+
+Especially software in traditional compiled programming languages consist in two forms: the *source code* and the *compiled code*. The transformation between the human readable source code and the machine readable compiled code is performed by a compiler. This is an irreversible process. The existence of both source and compiled code must be taken into account in the license, this makes data collection licenses unsuitable for software and software licenses unsuitable for data collections <sup id="datacommonsfaq-footnote">[DataCommons-faq](#datacommonsfaq)</sup>.
```
Comment (danielskatz) [inline README.md:50]: Regarding the first two sentences, data could also be available in 2 forms, though perhaps an irreversible transformation process would make less sense.

Regarding the third sentence, the license citation you give doesn't seem to be related to the fact that there is source code and compiled code. Rather, it is yet a third case (that I'm not sure I want to bring it up in this document) related to the discussion in "Software is a creative work, data are facts or observations" where we already talk about different licenses for software and data.  This reference says that neither of these types of licenses are right for data collections (basically databases containing data objects).  I can't see how this helps, either demonstrating the point in the first 2 sentences or in the overall document.

File: README.md [hunk 3]
```diff
@@ -43,34 +43,35 @@ Evidence: Citations
...
+Furthermore subtle differences between different compilers (e.g. on two different machines or two different operating systems) can result in subtly different results when the software is run on the same data. <sup id="floatcalc-footnote">[Float Differences](#floatcalc)</sup> This puts strong demands on the description of provenance. <sup id="swprov-footnote">[Software Running Provenance](#swprov)</sup>
```
Comment (danielskatz) [inline README.md:51]: I completely agree with the points you make here, but again I don't see what the difference between software and data is.  This says reproducibility is a hard problem, and that running the same software on the same data on a different system/environment can lead to a different result.  How does this show a difference between software and data?

**Review by danielskatz — CHANGES_REQUESTED** [review]
I appreciate your work in adding citations, but they do not convince me that your additional paragraph contributes to the differences between software and data.  I've added some detailed comments.  Please take a look and try to help me see why this addition makes the overall document stronger.

**Comment by rwwh** [comment]
OK. Thanks, and good luck with the project.

**Comment by danielskatz** [comment]
I would still like to add you as an author, since you contributed this discussion.  Is this ok with you?

**Comment by band** [comment]
Yes and thank you.

**Comment by danielskatz** [comment]
Sorry, this was implicitly addressed to @rwwh since he was the person who suggested this change and did the work to explain it.

@band, you are already listed as an author based on a previous contribution.

**Comment by band** [comment]
Ha ha ha. Gotta love a user's experience. I thought about the implicit recipient, but the web git UI fooled me.

**Comment by rwwh** [comment]
Yes and thank you. ;-) ?

I really appreciate this project and certainly appreciate being able to attach my name to it.

**Pull request closed without merging**